commit 603c3d8a8b79fbd15d95a9d36dd3ccefbdae9a3b
Author: Michael Pobega <mpobega@neverware.com>
Date:   Fri Mar 17 00:48:32 2017 +0000

    ACPI / blacklist: Make Dell Latitude 3350 ethernet work
    
    [ Upstream commit 708f5dcc21ae9b35f395865fc154b0105baf4de4 ]
    
    The Dell Latitude 3350's ethernet card attempts to use a reserved
    IRQ (18), resulting in ACPI being unable to enable the ethernet.
    
    Adding it to acpi_rev_dmi_table[] helps to work around this problem.
    
    Signed-off-by: Michael Pobega <mpobega@neverware.com>
    [ rjw: Changelog ]
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>
    
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/acpi/blacklist.c b/drivers/acpi/blacklist.c
index 4f87fd748303..4421f7c9981c 100644
--- a/drivers/acpi/blacklist.c
+++ b/drivers/acpi/blacklist.c
@@ -176,6 +176,18 @@ static struct dmi_system_id acpi_rev_dmi_table[] __initdata = {
 		      DMI_MATCH(DMI_PRODUCT_NAME, "Precision 3520"),
 		},
 	},
+	/*
+	 * Resolves a quirk with the Dell Latitude 3350 that
+	 * causes the ethernet adapter to not function.
+	 */
+	{
+	 .callback = dmi_enable_rev_override,
+	 .ident = "DELL Latitude 3350",
+	 .matches = {
+		      DMI_MATCH(DMI_SYS_VENDOR, "Dell Inc."),
+		      DMI_MATCH(DMI_PRODUCT_NAME, "Latitude 3350"),
+		},
+	},
 #endif
 	{}
 };
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1448_603c3d8a8b79fbd15d95a9d36dd3ccefbdae9a3b.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1448_603c3d8a8b79fbd15d95a9d36dd3ccefbdae9a3b.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
